packages/tasks/src/model-libraries.ts

```diff
@@ -150,6 +150,18 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"adapter_config.json"`,
 	},
+	deepforest_tree: {
```

Comment:
Can we collapse, `deepforest_tree`, `deepforest_bird` into one tag `deepforest`?

This way, for all the different model types you'd be able to add automatically track downloads.